<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>teacher</class>
 <widget class="QMainWindow" name="teacher">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>450</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>600</width>
    <height>450</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>600</width>
    <height>450</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>teacher</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="minimumSize">
    <size>
     <width>600</width>
     <height>450</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>600</width>
     <height>450</height>
    </size>
   </property>
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>6</x>
      <y>9</y>
      <width>591</width>
      <height>441</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>课表查询</string>
     </attribute>
     <widget class="QTableWidget" name="timetable">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>561</width>
        <height>251</height>
       </rect>
      </property>
      <property name="contextMenuPolicy">
       <enum>Qt::CustomContextMenu</enum>
      </property>
      <property name="verticalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOff</enum>
      </property>
      <property name="horizontalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOff</enum>
      </property>
      <property name="autoScroll">
       <bool>true</bool>
      </property>
      <property name="editTriggers">
       <set>QAbstractItemView::NoEditTriggers</set>
      </property>
      <property name="selectionMode">
       <enum>QAbstractItemView::NoSelection</enum>
      </property>
      <property name="gridStyle">
       <enum>Qt::DashDotDotLine</enum>
      </property>
      <property name="wordWrap">
       <bool>false</bool>
      </property>
      <property name="rowCount">
       <number>4</number>
      </property>
      <property name="columnCount">
       <number>5</number>
      </property>
      <attribute name="horizontalHeaderVisible">
       <bool>false</bool>
      </attribute>
      <attribute name="horizontalHeaderMinimumSectionSize">
       <number>25</number>
      </attribute>
      <attribute name="verticalHeaderVisible">
       <bool>false</bool>
      </attribute>
      <row/>
      <row/>
      <row/>
      <row/>
      <column>
       <property name="text">
        <string>星期一</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>星期二</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>星期三</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>星期四</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>星期五</string>
       </property>
      </column>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>290</y>
        <width>54</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 14pt &quot;华文彩云&quot;;</string>
      </property>
      <property name="text">
       <string>周数：</string>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBox">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>290</y>
        <width>69</width>
        <height>22</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>无</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>1</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>2</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>3</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>4</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>5</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>6</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>7</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>8</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>9</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>10</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>11</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>12</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>13</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>14</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>15</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>16</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>17</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>18</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>19</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>20</string>
       </property>
      </item>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>代课要求</string>
     </attribute>
     <widget class="QTableWidget" name="tableWidget">
      <property name="geometry">
       <rect>
        <x>5</x>
        <y>11</y>
        <width>571</width>
        <height>171</height>
       </rect>
      </property>
      <property name="selectionMode">
       <enum>QAbstractItemView::SingleSelection</enum>
      </property>
      <property name="selectionBehavior">
       <enum>QAbstractItemView::SelectRows</enum>
      </property>
      <property name="rowCount">
       <number>10</number>
      </property>
      <property name="columnCount">
       <number>10</number>
      </property>
      <attribute name="verticalHeaderVisible">
       <bool>false</bool>
      </attribute>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <column>
       <property name="text">
        <string>课程</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>1.星期几</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>1.大节</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>1.教室</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>2.星期几</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>2.大节</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>2.教室</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>3.星期几</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>3.大节</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>3.教室</string>
       </property>
      </column>
     </widget>
     <widget class="QComboBox" name="courseBox">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>200</y>
        <width>191</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>202</y>
        <width>54</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 14pt &quot;华文彩云&quot;;</string>
      </property>
      <property name="text">
       <string>课程：</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>260</y>
        <width>54</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 14pt &quot;华文彩云&quot;;</string>
      </property>
      <property name="text">
       <string>星期：</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>310</y>
        <width>54</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 14pt &quot;华文彩云&quot;;</string>
      </property>
      <property name="text">
       <string>大节：</string>
      </property>
     </widget>
     <widget class="QComboBox" name="dayBox1">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>260</y>
        <width>69</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QComboBox" name="timeBox1">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>310</y>
        <width>69</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QComboBox" name="dayBox2">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>260</y>
        <width>69</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QComboBox" name="dayBox3">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>260</y>
        <width>69</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QComboBox" name="timeBox3">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>310</y>
        <width>69</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QComboBox" name="timeBox2">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>310</y>
        <width>69</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>360</y>
        <width>54</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 14pt &quot;华文彩云&quot;;</string>
      </property>
      <property name="text">
       <string>教室：</string>
      </property>
     </widget>
     <widget class="QComboBox" name="classBox1">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>360</y>
        <width>69</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QComboBox" name="classBox2">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>360</y>
        <width>69</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QComboBox" name="classBox3">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>360</y>
        <width>69</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="apply">
      <property name="geometry">
       <rect>
        <x>450</x>
        <y>352</y>
        <width>75</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 14pt &quot;华文彩云&quot;;</string>
      </property>
      <property name="text">
       <string>申请</string>
      </property>
     </widget>
     <widget class="QPushButton" name="DelReqBtn">
      <property name="geometry">
       <rect>
        <x>450</x>
        <y>292</y>
        <width>75</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 14pt &quot;华文彩云&quot;;</string>
      </property>
      <property name="text">
       <string>删除</string>
      </property>
     </widget>
     <widget class="QPushButton" name="clearReqBtn">
      <property name="geometry">
       <rect>
        <x>450</x>
        <y>232</y>
        <width>75</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 14pt &quot;华文彩云&quot;;</string>
      </property>
      <property name="text">
       <string>清空</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
